naptime/src/main/scala/org/coursera/naptime/ari/engine/EngineImpl.scala

```diff
@@ -0,0 +1,20 @@
+package org.coursera.naptime.ari.engine
+
+import com.linkedin.data.schema.DataSchema
+import org.coursera.naptime.ari.EngineApi
+import org.coursera.naptime.ari.FetcherApi
+import org.coursera.naptime.ari.Request
+import org.coursera.naptime.ari.Response
+import org.coursera.naptime.schema.Resource
+
+import scala.concurrent.Future
+
+class EngineImpl(
+    override val schemas: Seq[Resource],
```

Comment:
i've been using naming where this would be called `resources` and the map of string->dataschema would be called `schemas`. i'm fine with either, but let's just make sure to be consistent.